docs/tutorials/storage.rst

```diff
@@ -0,0 +1,202 @@
+    :align: center
+
+    Pipeline Visualize MNIST
+
+The operation to visualize images called `VisualizeMNISTImages` is shown in the figure below:
+
+.. figure:: visualize_operation.png
+    :align: center
+
+    Operation VisualizeMNISTImages
+
+The code for the operation is shown below:
+
+.. code-block:: python
+
+    # Editing "VisualizeMNISTImages" Implementation
+    #
+    # The 'execute' method will be called when the operation is run
+    from matplotlib import pyplot
+
+    class VisualizeMNISTImages():
+
+        def execute(self, dataset):
+            images = dataset['X']
+            for i in range(9):
+                # define subplot
+                pyplot.subplot(330 + 1 + i)
+                # plot raw pixel data
+                pyplot.imshow(images[i], cmap=pyplot.get_cmap('gray'))
+            # show the figure
+            pyplot.show()
+
+Any `argument` to the execute method for an operation can use the artifacts available in a DeepForge project. To use the `mnist_sample` for visualizing the images in the pickle file add an `Input` node to the pipeline:
+
+.. figure:: input_node.png
+    :align: center
+
+    Input Node
+
+Replace the artifact in the added `Input` node and select `mnist_sample` as shown in the figure below:
+
+.. figure:: upload_artifact_to_input.png
+    :align: center
+
+    Using uploaded data in a pipeline
+
+After that, the data will be available to use in the operation, The following figure shows sample execution:
+
+.. figure:: sample_execution.png
+    :align: center
+
+    Visualizing Images
+
+
+Visualizing a FITS file with custom serialization
+-------------------------------------------------
+
+By default, DeepForge uses pickle to serialize `artifacts` and load them on operations. In this example, we will look into how to visualize fits images using a custom serialization in DeepForge. The FITS files we will use are available to download `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_r.fits.bz2>`_, `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_g.fits.bz2>`_ and `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_i.fits.bz2>`_. Before we start uploading the FITS files, we have to write a custom serializer for the FITS file, which can then be utilized for consumption in our operation.
+
+For visualizing FITS files, we will be using the `astropy <https://github.com/astropy/astropy>`_ python library. Function ``astropy.io.fits.open``, is able to handle a raw compressed binary FITS file or a simple filename. To visualize the file in DeepForge you can register a serializer for FITS file by writing a new class for handling them. For that, navigate to `My Utilities` and add a new python module called ``fits_file.py`` as highlighted in the figure below:
+
+.. figure:: custom_utils.png
+    :align: center
+
+    Custom Utils
+
+After you click on Enter, a text editor will open. Copy and paste the following content:
```

Comment:
I generally would not recommend saying things like "Copy and paste the following content" as the user can probably figure it out... It's generally better to stay focused on the content of the project rather then the ways that people follow along.